src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -66,8 +67,7 @@ static async Task RunAsync(ServerConfiguration serverConfiguration, Cancellation
         }
         else
         {
-            var timeout = TimeSpan.FromMinutes(1);
-            logger.LogCritical($"Server started with process ID {Environment.ProcessId}");
+            var timeout = TimeSpan.FromMinutes(2);
```

Comment:
How to attach debugger on a non-win platform?🤔 Use C# extenstions to debug itself?

Reply:
Yup - I do it often on my mac.  There's an attach to process command to attach to any dotnet core process